model/src/logger.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+export class Logger {
+    logger: any;
+    name: string;
+
+    constructor (logger: any, name: string) {
+      this.logger = logger
+      this.name = name
+    }
+
+    error = (message: string) => this.log('error', message)
```

Comment:
being pedantic, if you decide to keep the Logger as a class, you should change these to normal methods (not arrow syntax). With arrow syntax they're not getting added to the object prototype, so every time there is a new instance of Logger it will assign `error = (..` each time

Reply:
That's a thing I didn't know, thanks, I'll fix for now!